src/injector.js

```diff
@@ -0,0 +1,103 @@
+const puma = require('pumascript');
+const http = require('http');
+const fs = require('fs');
+
+
+let errorsLogs = [];
+
+
+function readDependencyJsonFile(dir) {
+    try {
+        return JSON.parse(fs.readFileSync(dir, 'utf8'));
+    } catch (e) {
+        console.log('The file entered does not exist');
```

Comment:
You can have parsing errors too. For this you should use: 

`Error parsing the file ${JSON.stringify(error)}`

Reply:
Done.